server/student.js

```diff
@@ -0,0 +1,75 @@
+import { Router } from "express";
+import query from "./db.js";
+import logger from "./utils/logger";
+
+const router = Router();
+
+router.post('/table', (req, res) => {
+	query
+		.query('CREATE TABLE IF NOT EXISTS students (id serial PRIMARY KEY, firstName VARCHAR(50) NOT NULL, lastName VARCHAR(50) NOT NULL, dateOfBirth DATE NOT NULL, email VARCHAR(150) NOT NULL, phoneNumber INTEGER  NOT NULL, passwordStudent VARCHAR (10))')
+		.then((result) => res.status(200).json(result.command))
+		.catch((error) => console.error(error));
+});
+router.post('/new', (req, res) =>{
+	const {firstName, lastName, dateOfBirth, email, phoneNumber, password} = req.body;
+	const Query = 'INSERT INTO students (firstName, lastName, dateOfBirth, email, phoneNumber, passwordStudent ) VALUES($1, $2, $3, $4, $5, $6)'
+	query
+		.query(Query, [firstName, lastName, dateOfBirth, email, phoneNumber, password])
+		.then(() => res.status(200).json({message: 'Student loaded successfully'}))
+		.catch((error) => res.status(500).json({error: 'Information loaded incorrectly - ' + error.message}));
```

Comment:
Maybe you can use a 400 here as a 500 error is reserved for server crashes https://developer.mozilla.org/en-US/docs/Web/HTTP/Status